### PR TITLE
Add get_all_equipment() method

### PR DESCRIPTION
**Pull request by canteli**
Added a method to the client that queries all the equipment from the buildings available for the organization.

### CODE REVIEW

File: onboard/client/client.py [hunk 1]
```diff
@@ -91,6 +91,15 @@ def get_all_points(self) -> List[int]:
             for e in equipment:
                 point_ids += e['points']
         return point_ids
+    
+    def get_all_equipment(self) -> List[int]:
```
Comment (gnmerritt) [inline onboard/client/client.py]: the type annotation here doesn't look correct

**Review by gnmerritt — CHANGES_REQUESTED** [review]

File: onboard/client/client.py (hunk 1 above)
Comment (canteli) [reply onboard/client/client.py]: Should it be List[dict] ?
I used the same type annotation as in the method " get_all_points(self) -> List[int] ", which returns a similar output (a list of dictionaries).

Comment (canteli) [reply onboard/client/client.py]: I can fix the type annotation of get_all_points too if that one should be List[Dict]

Comment (gnmerritt) [reply onboard/client/client.py]: ah yeah that'd be great - thank you. I'm going to file a ticket to add automated checks for these, too

Comment (canteli) [reply onboard/client/client.py]: I just fixed it, please let me know if that's good now

**Review by gnmerritt — APPROVED** [review]

**Pull request merged**